scripts/solr_builder/Jenkinsfile

```
@@ -16,9 +16,11 @@ pipeline {
     booleanParam(name: 'REUSE_POSTGRES', defaultValue: false, description: 'If true, assumes a functioning postgres is running/correct')
     booleanParam(name: 'REUSE_SOLRBACKUP', defaultValue: false, description: 'If true, assumes a functioning solr-backup is running/correct')
     booleanParam(name: 'INTO_SOLR_FROM_DUMP', defaultValue: true, description: 'If true, reindexes dump into solr')
-    booleanParam(name: 'INDEX_WORKS_ORPHANS', defaultValue: true, description: 'If true, reindexes works/orphans into solr')
+    booleanParam(name: 'INDEX_WORKS', defaultValue: true, description: 'If true, reindexes works into solr')
+    booleanParam(name: 'INDEX_ORPHANS', defaultValue: true, description: 'If true, reindexes orphans into solr')
     booleanParam(name: 'INDEX_AUTHORS', defaultValue: true, description: 'If true, reindexes authors into solr')
     booleanParam(name: 'INTO_SOLR_COPY_SUBJECTS', defaultValue: true, description: 'If true, copies subjects into solr')
+    string(name: 'MAX_CORES', defaultValue: '4', description: 'Max number of simultaneous cores')
```

This is what I actually use
```suggestion
    string(name: 'MAX_CORES', defaultValue: '12', description: 'Max number of simultaneous cores')
```